lib/text/vtt_text_parser.js

```diff
@@ -955,10 +955,13 @@ shaka.text.VttTextParser = class {
       '&gt;': '>',
       '&quot;': '"',
       '&#39;': '\'',
+      '&nbsp;': ' ',
+      '&lrm;': '',
```

Comment:
Left-to-right mark in unicode is '\u{200e}'

Reply:
Done!